metadata.go

```diff
@@ -11,78 +11,75 @@ type DataStatus = uint16
 
 const (
 	// DataStructureSet represents the data structure set flag
-	DataStructureSet DataStructure = iota
+	DataStructureSet DataStructure = 0
 
 	// DataStructureSortedSet represents the data structure sorted set flag
-	DataStructureSortedSet
+	DataStructureSortedSet DataStructure = 1
 
 	// DataStructureBTree represents the data structure b tree flag
-	DataStructureBTree
+	DataStructureBTree DataStructure = 2
 
 	// DataStructureList represents the data structure list flag
-	DataStructureList
-
-	// DataStructureNone represents not the data structure
-	DataStructureNone
+	DataStructureList DataStructure = 3
 )
 
 const (
 	// DataDeleteFlag represents the data delete flag
-	DataDeleteFlag uint16 = iota
+	DataDeleteFlag DataFlag = iota
 
 	// DataSetFlag represents the data set flag
-	DataSetFlag
+	DataSetFlag DataFlag = 1
 
 	// DataLPushFlag represents the data LPush flag
-	DataLPushFlag
+	DataLPushFlag DataFlag = 2
 
 	// DataRPushFlag represents the data RPush flag
-	DataRPushFlag
+	DataRPushFlag DataFlag = 3
 
 	// DataLRemFlag represents the data LRem flag
-	DataLRemFlag
+	DataLRemFlag DataFlag = 4
 
 	// DataLPopFlag represents the data LPop flag
-	DataLPopFlag
+	DataLPopFlag DataFlag = 5
 
 	// DataRPopFlag represents the data RPop flag
-	DataRPopFlag
+	DataRPopFlag DataFlag = 6
 
 	// DataLTrimFlag represents the data LTrim flag
-	DataLTrimFlag
+	DataLTrimFlag DataFlag = 7
```

Comment:
Since the `DataLSetFlag` equal to 7 was removed, `DataLTrim` needs to be started from 8, meaning that subsequent flag values need to be increased by 1

Reply:
You can switch to the old version and copy it 🤣

Reply:
Roger that